wallet-integration/guide.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Wallet Integration Guide
+sidebarTitle: Guide
+description: This guide will walk you through setting up the ChainPatrol API into your wallet.
+---
+
+We're going to assume you're creating a web-based wallet, but the same
```

Comment:
@umar-ahmed I did a first pass on this guide. I want to add screenshots but let me know if this flow makes sense and any suggestions before I continue